typescript/interchain/contracts/InterchainAccountRouter.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.13;
+
+import {OwnableMulticall, Call} from "./OwnableMulticall.sol";
+
+// ============ External Imports ============
+import {Router} from "@abacus-network/app/contracts/Router.sol";
+import {TypeCasts} from "@abacus-network/core/contracts/libs/TypeCasts.sol";
+import {Create2} from "@openzeppelin/contracts/utils/Create2.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+
+/*
+ * @title The Hello World App
```

Comment:
rm?

Reply:
skipping to avoid changing bytecode metadata from deployed